### DISCUSSION TITLE
Installing rpm on mac os 14.0

### DISCUSSION THREAD

**Discussion by ddbrodsky** [Homebrew/discussions#4828] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.14
ORIGIN: https://github.com/Homebrew/brew
HEAD: affc4c01aada2c973b63e084e7696e896edf2b7b
Last commit: 4 days ago
Core tap JSON: 05 Oct 23:27 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: vi
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.42.0 => /opt/homebrew/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 14.0-arm64
CLT: 15.0.0.0.1.1694021235
Xcode: 15.0
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Homebrew's "sbin" was not found in your PATH but you have installed
formulae that put executables in /opt/homebrew/sbin.
Consider setting your PATH for example like so:
  echo 'export PATH="/opt/homebrew/sbin:$PATH"' >> ~/.profile
```

### Description of issue

I am trying to install or reinstall rpm, I used to have it up and running and working with mac os 13.x, but it seems to have broken with 14.0 or sometime recently.  When I try to install it, it tells me: 

==> Downloading https://formulae.brew.sh/api/formula.jws.json
#=#=- #       #
==> Downloading https://formulae.brew.sh/api/cask.jws.json
#=#=- #       #
rpm: Linux is required for this software.
attr: Linux is required for this software.
acl: Linux is required for this software.
elfutils: Linux is required for this software.
libcap: Linux is required for this software.
Error: rpm: Unsatisfied requirements failed this build.

Was it recently converted to linux only?  is it possible to get rpm back?

Thanks!

**Comment by ZhongRuoyu** [comment]
> Was it recently converted to linux only? is it possible to get rpm back?

Yes, it was intentional. Please see #4806 and #4826 for related discussions.
